### PR TITLE
[49245] Changed dates are misaligned with the other dates

### PR DESCRIPTION
**Pull request by bsatarnejad**
https://community.openproject.org/projects/openproject/work_packages/49245/activity

### CODE REVIEW

**Review by HDinger — CHANGES_REQUESTED** [review]
The due date is now moved to the right, although all other dueDate fields are not:

Bildschirmfoto 2023-07-21 um 07 59 27

Further, please extract the value to a global variable and use that for the indentation of the normal display field as well. Otherwise we will probably forget about one or the other when we want to change the value in the future.

File: frontend/src/global_styles/layout/work_packages/_table_baseline.sass [hunk 1]
```diff
@@ -56,6 +56,11 @@
     // Ensure items are not taking full width
     align-items: flex-start
 
+    &.startDate
+      padding-left: $work-package--start-date-display-field-padding-left
+
+  .spot-icon
+    vertical-align: middle
```
Comment (HDinger) [inline frontend/src/global_styles/layout/work_packages/_table_baseline.sass]: Out of curiosity: Why did you add this rule? It doesn't make any difference for me locally. 🤔

File: frontend/src/global_styles/openproject/_variables.sass [hunk 2]
```diff
@@ -4,3 +4,5 @@ $secondary-color: #bfbfbf
 
 // A selector that checks whether we are running in a test environment
 $spec-active-selector: '.env-test'
+
+$work-package--start-date-display-field-padding-left: 1.5rem
```
Comment (HDinger) [inline frontend/src/global_styles/openproject/_variables.sass:4]: Please use the same variable also for the indentation of the header element so that we don't run into problems when changing the value.

**Review by HDinger — CHANGES_REQUESTED** [review]
I have only minor remarks (see comments in code)

File: frontend/src/global_styles/layout/work_packages/_table_baseline.sass (hunk 1 above)
Comment (bsatarnejad) [reply frontend/src/global_styles/layout/work_packages/_table_baseline.sass]: Hey @HDinger I think it is due to a merge from outdated dev branch. I'll remove it.

**Review by HDinger — APPROVED** [review]

**Comment by HDinger** [comment]
Merging as the test is unrelated
